### ISSUE TITLE
Node & Npm requirements

### ISSUE THREAD

**Issue by Shuunen** [pingyhq/pingy-cli#11]
Hi,

I installed your cli with node v4.6.0 & npm v4.4.4 and the init failed : 
```
pingy init
/home/me/.nvm/versions/node/v4.6.0/lib/node_modules/@pingy/cli/cli-run.js:12
const { inspect } = require('util');
      ^
SyntaxError: Unexpected token {
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:373:25)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at run (/home/me/.nvm/versions/node/v4.6.0/lib/node_modules/@pingy/cli/cli.js:14:15)
    at Object.<anonymous> (/home/me/.nvm/versions/node/v4.6.0/lib/node_modules/@pingy/cli/cli.js:26:3)
    at Module._compile (module.js:409:26)
```

As you can see i'm using nvm, so I tried to install you cli on a more recent node & npm versions, and the init worked well :+1: 

So I think you should specify the minimum node & npm versions to use your cli to avoid loosing users/developers/customers, some people can stop using it after pingy init failed.

**Comment by davej** [comment]
It's specified in the [package.json](https://github.com/pingyhq/pingy-cli/blob/master/packages/cli/package.json) but unfortunately npm only gives a warning which is easy to miss. 

I'll try to make the version information more prominent on the site and perhaps output a better error in the code too. Thanks!

**Comment by Shuunen** [comment]
Cool :+1: 

By the way thanks for your cli, it's really cool/fast/light to use and a good idea compared to the million heavyweight solutions around there :1st_place_medal:

**Comment by davej** [comment]
Awesome to hear. Spread the word 🙂

**Comment by davej** [comment]
Fixed in `v0.6.5`. Thanks!

<img width="605" alt="screen shot 2017-05-11 at 16 29 23" src="https://cloud.githubusercontent.com/assets/26971/25957836/1e5deaa4-3667-11e7-988b-d35b12f95183.png">
<img width="497" alt="screen shot 2017-05-11 at 15 37 42" src="https://cloud.githubusercontent.com/assets/26971/25957844/2119689a-3667-11e7-80e5-95c36314f582.png">
